app/lib/travel_planner/journey.rb

```diff
@@ -24,7 +24,12 @@ def format_legs(journey_data)
     journey = journey_data['Leg'].map do |leg_data|
       leg = TravelPlanner::Leg.new leg_data, @coords
 
-      formatted_leg = leg.type=='BIKE' ? format_bike(leg) : format_public(leg)
+      formatted_leg = case leg.type
+      when 'BIKE', 'WALK'
+        format_bike(leg)
```

Comment:
That is for the route planner to decide - but now we are at least asking it, if the Rejseplanen has a leg where they think the user should walk